1.20.4-forge/src/main/resources/META-INF/mods.toml

```diff
@@ -0,0 +1,33 @@
+modLoader = "javafml"
+loaderVersion = "[0,)"
+license = "LGPL-3.0-or-later"
+issueTrackerURL = "https://github.com/The-Fireplace-Minecraft-Mods/In-Game-Account-Switcher/issues"
+clientSideOnly = true
+
+[[mods]]
+modId = "ias"
+version = "${version}"
+displayName = "In-Game Account Switcher"
+updateJSONURL = "https://raw.githubusercontent.com/The-Fireplace-Minecraft-Mods/In-Game-Account-Switcher/main/updater-forge.json"
+displayURL = "https://modrinth.com/mod/in-game-account-switcher"
+logoFile = "ias.png"
+credits = "The_BrainStone, 2xsaiko, Elvm"
+authors = "The_Fireplace, VidTu"
```

Comment:
You can go ahead and list yourself as the main author and put me in the credits, same on the fabric.mod.json

Reply:
done